startup/interface-rename-sideway.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
```

Comment:
While I would also like to see this horrible hack removed but I guess it makes sense to do it at the same time as similar changes are made post-installation (CP-43296).